tests/test_arrow_eager.py

```diff
@@ -989,7 +989,7 @@ def test_batch_fixed_lists(self):
 
         self.run_test_case(dataset, truth_data, batch_size=batch_size)
 
-    def test_batch_variable_length_list(self):
+    def test_batch_variable_length_list_batched(self):
         """Test batching with variable length lists raises error
     """
```

Comment:
@BryanCutler Since this file has been touched, would you mind to align the docstring as well? The misalign of docstring is part of the leftovers caused by black, see  issue #861 for more details.

With respect to this file, as you could see, 
```
    def test_batch_variable_length_list(self):
         """Test batching with variable length lists raises error
     """
```

should be (the second `"""` should intent two more spaces.
```
    def test_batch_variable_length_list(self):
         """Test batching with variable length lists raises error
        """
```

Reply:
Sure, I wasn't sure how the formatting was supposed to be